src/main/java/com/codessquad/qna/question/QuestionController.java

```diff
@@ -0,0 +1,54 @@
+package com.codessquad.qna.question;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@Controller
+@RequestMapping("questions")
```

Comment:
항상 슬래시(`/`) 를 잊지 말아주세요. 불필요한 혼동을 방지하기 위해서요. 

```suggestion
@RequestMapping("/questions")

Reply:
fc95d84
추가했습니다.
혼동을 줄 수 있는 이유가 어노테이션에 지정된 url을 스프링 내부(디스패쳐 서블릿?)에 저장할때 슬래쉬를 자동으로 붙여주는 것 같았는데,
이러면 절대경로인지 상대경로인지 명시적으로 작성되지 않아서 헷갈릴 수 있다고 생각되었습니다. 혹시 제가 생각한 부분이 맞나요?